sources/packages/web/src/services/http/dto/EducationProgramOffering.dto.ts

```diff
@@ -132,6 +132,7 @@ export interface EducationProgramOfferingSummaryAPIOutDTO {
 
 export interface OfferingStartDateAPIOutDTO {
```

Comment:
This DTO's name doesn't reflect what its data now.

Reply:
changed.